ietf/doc/views_conflict_review.py

```diff
@@ -375,6 +377,23 @@ class SimpleStartReviewForm(forms.Form):
                              label="Notice emails", 
                              help_text="Separate email addresses with commas.", 
                              required=False)
+    
+    def clean(self):
+        cleaned_data = super().clean()
+        if 'notify' not in cleaned_data:
+            return
+                
+        notify = cleaned_data['notify']
+        stream_managers = [
+            re.sub('([^<]*<)?(.*)(>.*)', '\2', r) 
```

Comment:
Could use `email.utils.parseaddr()` here - it'll return a tuple, `(realname, email)`. If you apply that to addresses in the notifications list when filtering, I think that'd get rid of the caveat (although I 100% agree they get what they deserve if you choose to keep it simple)